pkg/tsdbctl/add.go

```diff
@@ -251,9 +251,6 @@ func (ac *addCommandeer) appendMetric(
 
 	ac.rootCommandeer.logger.DebugWith("Adding a sample value to a metric.", "lset", lset, "t", tarray, "v", varray)
 
-	if print {
```

Comment:
If you decided to drop this print then go ahead and drop the boolean `print` parameter from the `appendMetric` method.
I believe that this "debug" print was added by @yaronha to reflect the input stream content when using CLI with `stdin` or `file` input. Please consider adding useful log message if necessary (using logger API) to let user know what is going on under the hood (@debug level)